src/cmd/operator/main.go

```diff
@@ -52,18 +52,17 @@ func main() {
 	version.LogVersion()
 
 	namespace := os.Getenv("POD_NAMESPACE")
-	cfg, err := config.GetConfig()
-	if err != nil {
-		log.Error(err, "")
-		os.Exit(1)
-	}
-
 	var mgr manager.Manager
 	var cleanUp func()
 
 	subCmd := getSubCommand()
 	switch subCmd {
 	case operatorCmd:
+		cfg, err := config.GetConfig()
```

Comment:
can this be done in a more reusable way? same 5 lines in 3 of the 4 sub commands...